src/app/components/PageLayoutWrapper/index.tsx

```diff
@@ -40,15 +35,19 @@ const PageLayoutWrapper = ({
   status,
 }: PropsWithChildren<Props>) => {
   const { service } = useContext(ServiceContext);
-  const { isAmp, isNextJs, variant } = useContext(RequestContext);
+  const {
+    isAmp,
+    isNextJs,
+    variant,
+    pageType: fallbackPageType,
+  } = useContext(RequestContext);
 
   const scriptSwitchId = pathOr('', ['scriptSwitchId'], pageData);
   const renderScriptSwitch = pathOr(true, ['renderScriptSwitch'], pageData);
 
   const isErrorPage = ![200].includes(status) || !status;
   const metadataType = pageData?.metadata?.type;
-  const otherPageType = pageData?.pageType;
-  const pageType = metadataType || otherPageType;
+  const pageType = metadataType || fallbackPageType;
```

Comment:
Hmm yes... if we have a fallback page type, then we could end up reporting a different page type than one we're expecting. However, since we have not really changed the logic, the existing page types sent to Web Vitals should still work as they did before.

Having the fallback in place means that any future / new page types (e.g. Home, Live) don't need to have the page type set on `pageData.metadata.type`, and can just use the pageType from the request context. 

The PageType is based on the url i.e. /kyrgyz is a HomePage, /kyrgyz/topics/12345 is a TopicPage, and this will always be populated, since it's set at the highest level when we make the request to Simorgh.

Reply:
Where is `fallbackPageType` defined? I'd say if the type parameter is intended to be mandatory we might want to keep the code simple and just fail to report web vitals? I feel like we'd only want a fallback for user critical functionality and this isn't.

My thinking around this is around reducing complexity and improving readability.

Reply:
`fallbackPageType` is a rename of `pageType` which comes from the Request Context.

I agree with keeping it simple and readable... so do you think we should remove the fallback functionality and instead ensure that any new pages have `metadata.type` set accordingly (which _should_ be the same as the `pageType` on `RequestContext`), so that we will see an error / warning message if there is no page type set?

Reply:
So, just to confirm:

- remove all references to `fallbackPageType` / getting page type from RequestContext
- set pageType to be `pageData.metadata.type`
  - update the props accordingly to make `metadata.type` mandatory not optional
- set `metadata.type` in homepage/getInitialData instead of page type (see https://github.com/bbc/simorgh/blob/c423e9eced5121a03e2701ad6721e0f0e2a9fc0f/src/app/routes/homePage/getInitialData/index.ts#L57), to ensure that HomePages report the correct page type in web vitals
- We are going to remove the check for pageType when rendering the WebVitals component below, so that we get error messages in the console if there is no page type to report
- Aside - we will no longer be reporting Web Vitals for Error Pages, as it is not very useful

Reply:
Looks good to me. There were some cases when working on investigating where the no pageType warnings were coming from where the home page route was giving us no pageData despite the mock data being present. We tried a combination of things and @eagerterrier found it was the route that was the issue. So as long as we keep in the web vitals component render condition for pageData being present _as well_ as if it is an error page, then should be ok to always check fro metadata.type when there is page data.